src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    {person.age && <p className="Person__age">I am {person.age}</p>}
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {person.partnerName} is my {person.sex === 'f' ? 'husband' : 'wife'}
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
```

Comment:
Move ternary inside of .Person__partner. Should only text rerender.
To simplify make two ternary in component and pass just constant of second ternary to jsx.
Something like that.
```
const partnerType = person.sex === 'f' ? 'husband' : 'wife'
const personPartner = person.isMarried ? `${person.partnerName} is my {personPartner}` : 'I am not married';
```

```suggestion
      <p className="Person__partner">{personPartner}</p>
```